pkg/base/BUILD.bazel

```diff
@@ -64,3 +65,9 @@ go_test(
         "@com_github_stretchr_testify//require",
     ],
 )
+
+# Using common function for stringer to create testclusterreplicationmode_string.go
```

Comment:
I don't think this comment is useful, just delete it at this point.

Reply:
I used it as there is no "real" clue on which will be the output (only that you go and check the function) but yes, you are right I can delete it in all of the files :)